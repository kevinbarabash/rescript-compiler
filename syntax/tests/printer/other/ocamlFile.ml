(* parses ocaml files *)
let x =
  let a = 1 in
  let b = 2 in
  a + b
