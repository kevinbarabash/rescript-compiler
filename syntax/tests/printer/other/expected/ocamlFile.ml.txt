/* parses ocaml files */
let x = {
  let a = 1
  let b = 2
  a + b
}
